dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -371,10 +382,9 @@ def runValidation(self, jobId, interfaces):
                             continue
 
                     if not passedValidations:
-                        if failures:
+                        if self.writeErrors(failures, interfaces, jobId, shortColnames, writer, warningWriter, rowNumber):
                             rowErrorPresent = True
                             errorRows.append(rowNumber)
```

Comment:
How do we want to treat the warning/error count for the new `number_of_rows_valid` column in the job table? If I'm reading this line correctly, we're not going to strip a row of it's valid status if it only has warnings (because we're not doing `errorRows.append(rowNumber)` unless there's a fatal error.

However, we're treating everything that comes back in the sql error query as a reason to count a row as invalid, regardless of its fatal/warning status (see line 398).

In the longer term, it probably makes sense to add additional metadata to the job table if people request it, but in the short term, we should decide whether or not we want to consider warnings when marking a row as valid.